### PR TITLE
Report body mass changes, fix plot labels, and add description for `final.osim`

### PR DESCRIPTION
**Pull request by nickbianco**
Addresses issues: #49, #79, #80 

### Summary of changes
- Fixed results plotting error where marker error time range was incorrectly scaled.
- Fixed incorrect plot labels for GRFs and IK and ID results.
- Added a description of the output model `final.osim` to the README.
- Added a breakdown of the changes in the model total and body segment masses to the README.

### CODE REVIEW

File: server/engine/plotting.py [hunk 1]
```diff
@@ -217,8 +217,8 @@ def generateReportForTable(table, filename, output_fpath, data_type, bilateral=T
     refs = list()
 
     # Suffixes to detect if a pelvis coordinate is translational or rotational.
-    translate_suffixes = ['_tx', '_ty', '_tz', '_force']
-    rotate_suffixes = ['_tilt', '_list', '_rotation', '_moment']
+    translate_suffixes = ['_tx', '_ty', '_tz', '_force', '_vx', '_vy', '_vz', '_px', '_py', '_pz']
+    rotate_suffixes = ['_tilt', '_list', '_rotation', '_moment', '_mx', '_my', '_mz', '_x', '_y', '_z']
```
Comment (carmichaelong) [inline server/engine/plotting.py:221]: Are these additional suffixes necessary? Looks like they were only used for columns containing `pelvis`. I could easily be missing something here, but seems like L273-291 and L304 may be covering these cases.

**Review by carmichaelong — CHANGES_REQUESTED** [review]
Overall looks great. Just one comment/question, where I might be overlooking something.

File: server/engine/plotting.py (hunk 1 above)
Comment (nickbianco) [reply server/engine/plotting.py]: Yes, I meant to remove these after coming up with a different solution. Updated.

**Comment by nickbianco** [comment]
@carmichaelong, ready for review.

**Review by carmichaelong — APPROVED** [review]

**Comment by nickbianco** [comment]
Thanks @carmichaelong.

**Pull request merged**